oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LucenePropertyIndex.java

```diff
@@ -1644,11 +1647,39 @@ public List<Facet> getFacets(int numberOfFacets, String columnName) throws IOExc
                 return cachedResults.get(cacheKey);
             }
             LOG.trace("columnName = {} facet Data not present in cache...", columnName);
+            if (EAGER_FACET_CACHE_FILL) {
+                fillFacetCache(numberOfFacets);
+                if (cachedResults.containsKey(cacheKey)) {
+                    LOG.trace("columnName = {} now found");
```

Comment:
`cacheKey` is missing
```suggestion
                    LOG.trace("columnName = {} now found", cacheKey);
```